doc/userguide/configuration/exception-policies.rst

```diff
@@ -279,16 +279,17 @@ be logged when exception policies are enabled:
    * - Setting
      - Counter
    * - stream.memcap
-     - tcp.ssn_memcap_exception_policy
+     - exception_policy.tcp.ssn_memcap
    * - stream.reassembly.memcap
-     - tcp.reassembly_memcap_exception_policy
+     - exception_policy.tcp.reassembly.memcap
    * - stream.midstream
-     - tcp.midstream_exception_policy
+     - exception_policy.tcp.midstream
    * - defrag.memcap
-     - defrag.memcap_exception_policy
+     - exception_policy.defrag.memcap
    * - flow.memcap
-     - flow.memcap_exception_policy
+     - exception_policy.flow.memcap
    * - app-layer.error
+     - exception_policy.app_layer.error.exception_policy
      - app_layer.error.exception_policy
```

Comment:
uuuuuuuuuuuuuuurgh, forgot to actually build the docs before submitting PR. that's what I get for it.

Reply:
force-pushed fixes for this.